.github/workflows/release.yml

```diff
@@ -26,12 +26,12 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install -U pip
-          python -m pip install -U setuptools twine wheel
+          python -m pip install -U twine build
 
       - name: Build package
         run: |
           python setup.py --version
-          python setup.py sdist --format=gztar bdist_wheel
+          python -m build
           twine check dist/*
```

Comment:
`--strict` wouldn't hurt

Reply:
In fact adding strict would require fixing another bug, that is why I did not mention it during review.